File: 4point5.py
def isBST(root):
	if(!subroot.left and !subroot.right):
		return true
	elif((subroot.left > subroot) or (subroot.right < subroot)):
		return false
	else:
		return (isBST(subroot.left) and isBST(subroot.right))

File: 4point5.java
boolean isBST(TreeNode root)
{
	if(!subroot.left && !subroot.right)
	{
		return true;
	}
	else if(subroot.left > subroot || subroot.right < subroot)
	{
		return false;
	}
	else
	{
		 return (isBST(subroot.left) && isBST(subroot.right));
	}
}